frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -56,11 +63,17 @@ const Header: React.FC<Properties> = ({ name, avatar, dataTabs }) => {
                         <Menu />
                     )}
                 </div>
-                {name ? (
+                {token ? (
                     <Link className={styles.userLink} to={AppRoute.USER}>
                         <div className={styles.headerLogo}>
                             <div className={styles.userLogo}>
-                                {avatar && <img src={avatar} alt="user" />}
+                                {avatar && (
+                                    <img
+                                        className={styles.imgLogo}
+                                        src={avatar}
+                                        alt="user"
+                                    />
```

Comment:
I would put it in a separate component, and would provide for the display of the user's avatar by default

Reply:
But you didn't create a separate component with a fallback. So could you do it?